lib/metrician/core_ext.rb

```diff
@@ -0,0 +1,48 @@
+begin
+  require 'active_support/core_ext/hash/deep_merge'
+rescue LoadError
+  class Hash
+    # active_support
+    def deep_merge(other_hash, &block)
+      dup.deep_merge!(other_hash, &block)
```

Comment:
It's the ruby standard dup method from Object: https://ruby-doc.org/core-2.4.1/Object.html#method-i-dup